recipes/gtdbtk/meta.yaml

```diff
@@ -48,6 +49,9 @@ about:
   license_file: 'LICENSE'
   summary: 'A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.'
 extra:
+  additional-platforms:
+  # requires mash and pplacer
+  #  - linux-aarch64
```

Comment:
The recipe is noarch. 
There is no need of additional platforms here

Reply:
We should lint this somehow.